chains.yaml

```diff
@@ -2469,3 +2469,26 @@ chain-settings:
           grpcId: 10129
           short-names: [ swell-testnet ]
           chain-id: 0x784
+    - id: kadena
+      label: Kadena
+      type: kadena
+      settings:
+        expected-block-time: 10s
```

Comment:
looks like should be smaller (cause it changes every new block in subchain arrived)